src/loader-components/_ExampleTemplate.jsx

```diff
@@ -30,93 +31,68 @@ const defaultProps = {
   isExpanded: false,
 };
 
-class ExampleTemplate extends React.Component {
-  static renderHeader(title) {
-    if (title) {
-      return (
-        <div className={cx('header')}>
-          <h2 className={cx('title')}>
-            {title}
-          </h2>
-        </div>
-      );
-    }
-    return null;
-  }
-
-  static renderDescription(description) {
-    if (description) {
-      return (
-        <div className={cx('description')}>
-          {description}
-        </div>
-      );
-    }
-    return null;
-  }
-
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      isExpanded: props.isExpanded,
-      isBackgroundTransparent: false,
-    };
-
-    this.handleBgToggle = this.handleBgToggle.bind(this);
-    this.handleCodeToggle = this.handleCodeToggle.bind(this);
-  }
-
-  handleBgToggle() {
-    this.setState(prevState => ({ isBackgroundTransparent: !prevState.isBackgroundTransparent }));
-  }
-
-  handleCodeToggle() {
-    this.setState(prevState => ({ isExpanded: !prevState.isExpanded }));
+const renderHeader = (title) => {
+  if (title) {
+    return (
+      <div className={cx('header')}>
+        <h2 className={cx('title')}>
+          {title}
+        </h2>
+      </div>
+    );
   }
+  return null;
+};
```

Comment:
This is personal preference, so you don't have to change it, but I'd inline these in the ExampleTemplate directly rather than create separate functions to return html snippets. This would remove the functions for `renderHeader` and `renderDescription`. 

```jsx
  return (
    <div className={cx('template', theme.className)}>
      {title && (
        <div className={cx('header')}>
          <h2 className={cx('title')}>
            {title}
          </h2>
        </div>
      )}
      <div className={cx('content', { 'dynamic-content': isBackgroundTransparent })}>
        {description && (
          <div className={cx('description')}>
            {description}
          </div>
        )}
        {example}
      </div>
      {exampleSrc
        && (
          <div className={cx('footer')}>
            <div className={cx('button-container')}>
              <button type="button" className={cx('bg-toggle')} onClick={() => setIsBackgroundTransparent(!isBackgroundTransparent)}>
                Toggle Background
              </button>
              <button type="button" className={cx('code-toggle')} onClick={() => setCodeIsVisible(!codeIsVisible)}>
                <span className={cx('chevron-left')} />
                <span>Code</span>
                <span className={cx('chevron-right')} />
              </button>
            </div>
            {codeIsVisible
              && (
                <div className={cx('code')}>
                  {exampleSrc}
                </div>
              )}
          </div>
        )}
    </div>
  );
```